test/listen-dial.spec.js

```diff
@@ -54,87 +61,84 @@ describe('listen', () => {
     })
   })
 
+  it('listen on path', async () => {
+    // Windows doesn't support unix paths
+    const mh = isWindows
+      ? multiaddr('/ip4/0.0.0.0/tcp/8080')
```

Comment:
should we just skip this test in windows instead?

Reply:
yeah, that make more sense